packages/babel-plugin-transform-es2015-spread/src/index.js

```diff
@@ -1,9 +1,8 @@
-export default function({ types: t }) {
-  function getSpreadLiteral(spread, scope, state) {
-    if (
-      state.opts.loose &&
-      !t.isIdentifier(spread.argument, { name: "arguments" })
-    ) {
+export default function({ types: t }, options) {
+  const { loose } = options;
+
+  function getSpreadLiteral(spread, scope) {
```

Comment:
Can remove passing down `state` at the call sites:

https://github.com/babel/babel/pull/6381/files#diff-656ce81f115b2958c89cf5869995f057R34